docs/events/api/timeline-template-api-update.asciidoc

```diff
@@ -0,0 +1,385 @@
+[[timeline-template-api-update]]
+=== Update Timeline or Timeline template
+
+Updates an existing Timeline or Timeline template.
+
+Use the `timeline` object's <<timeline-object-typeField, `timelineType`>> field
+to specify whether a Timeline or a Timeline template is updated, where:
+
+* `"timelineType": "default"` updates an existing Timeline
+* `"timelineType": "template"` updates an existing Timeline template
+
+==== Request URL
+
+`PATCH <kibana host>:<port>/api/timeline`
+
+==== Request body
+
+A JSON object defining the Timeline or Timeline template query and time filter.
+
+NOTE: For detailed information about the Timeline object schema and its
+corresponding UI components, see <<timeline-object-schema>>.
+
+[width="100%",options="header"]
+|==============================================
+|Name |Type |Description |Required
+
+|`timeline` |<<timeline-object-schema, timeline>> a|The timeline object, which
+defines the search criteria and time range.
+
+When you are updating a Timeline template, provide these fields so you can
```

Comment:
Since users might choose to only update a Timeline, it might make more sense to use `if` (which is more conditional) over `when` (which is more certain). 

I also suggested a minor edit to the instructional portion to make it real clear that users need to specify field values, not just the fields. Feel free to ignore my edit if it's too nit-picky. It might be safe to assume that API users will know to define the fields in their request.

```suggestion
If you're updating a Timeline template, provide values for these fields so you can
```

Reply:
Agree with the first point.
Re the second point, I think "fields" is sufficient, as we're also explaining in the following lines what those fields' values should be.